### DISCUSSION TITLE
Add images hosted on google photos?

### DISCUSSION THREAD

**Discussion by rastafrange** [facebook/docusaurus#3535] [Q&A] [answered]
Hi everyone,

I'd like to know whether I can add pictures hosted on a public google photos album? 
Indeed I intend to have some blog posts with many images and I wonder how to do that with docusaurus. 
Or maybe I could use git lfs?

Thanks for your feedback

**Comment by slorber** [comment, marked as answer]
Hi

You can put your images wherever you want as long as they are publicly available.
